static/menu.css

```diff
@@ -136,26 +132,26 @@ legend {
 }
 
 .inline-input input {
-  min-height: 1em;
+  min-height: 1rem;
```

Comment:
Need to make sure this inline input component has the same properties as the other input components, since they no longer use the min-height property. min-height does wonky stuff to input fields. Take a look at the tags page to verify.

Reply:
Nevermind, this is not what I thought this was :p